python/hail/docs/tutorials/07-filter-annotate.ipynb

```diff
@@ -0,0 +1,231 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "## Filter\n",
+    "\n",
+    "You can filter the rows of a table with [Table.filter](https://hail.is/docs/devel/hail.Table.html#hail.Table.filter).  You can filter on an expression, `filter` keeps those rows for which the expression evaluates to `True`.  `filter` returns another `Table`."
```

Comment:
You can filter the rows of a table using Boolean expression with filter. This returns a table of those rows for which the expression evaluates to `True`.

(you missed this the first time)